tests/MyGame/Example/Monster.java

```diff
@@ -344,5 +347,372 @@ public static Monster __lookup_by_key(Monster obj, int vectorLocation, String ke
+    AnyUniqueAliasesUnion _oAnyUnique = new AnyUniqueAliasesUnion();
+    byte _oAnyUniqueType = this.anyUniqueType();
+    _oAnyUnique.setType(_oAnyUniqueType);
+    Table _oAnyUniqueValue  = this.anyUnique();
+    switch (_oAnyUniqueType) {
+      case AnyUniqueAliases.M:
+        _oAnyUnique.setValue(_oAnyUniqueValue != null ? ((MyGame.Example.Monster) _oAnyUniqueValue).unpack() : null);
+        break;
+      case AnyUniqueAliases.TS:
+        _oAnyUnique.setValue(_oAnyUniqueValue != null ? ((MyGame.Example.TestSimpleTableWithEnum) _oAnyUniqueValue).unpack() : null);
+        break;
+      case AnyUniqueAliases.M2:
+        _oAnyUnique.setValue(_oAnyUniqueValue != null ? ((MyGame.Example2.Monster) _oAnyUniqueValue).unpack() : null);
+        break;
+      default: break;
+    }
+    _o.setAnyUnique(_oAnyUnique);
+    AnyAmbiguousAliasesUnion _oAnyAmbiguous = new AnyAmbiguousAliasesUnion();
+    byte _oAnyAmbiguousType = this.anyAmbiguousType();
+    _oAnyAmbiguous.setType(_oAnyAmbiguousType);
+    Table _oAnyAmbiguousValue  = this.anyAmbiguous();
+    switch (_oAnyAmbiguousType) {
+      case AnyAmbiguousAliases.M1:
+        _oAnyAmbiguous.setValue(_oAnyAmbiguousValue != null ? ((MyGame.Example.Monster) _oAnyAmbiguousValue).unpack() : null);
+        break;
+      case AnyAmbiguousAliases.M2:
+        _oAnyAmbiguous.setValue(_oAnyAmbiguousValue != null ? ((MyGame.Example.Monster) _oAnyAmbiguousValue).unpack() : null);
+        break;
+      case AnyAmbiguousAliases.M3:
+        _oAnyAmbiguous.setValue(_oAnyAmbiguousValue != null ? ((MyGame.Example.Monster) _oAnyAmbiguousValue).unpack() : null);
+        break;
+      default: break;
+    }
+    _o.setAnyAmbiguous(_oAnyAmbiguous);
+    List<Integer> _oVectorOfEnums = new ArrayList<>();
+    for (int _j = 0; _j < this.vectorOfEnumsLength(); ++_j) {_oVectorOfEnums.add(this.vectorOfEnums(_j));}
+    _o.setVectorOfEnums(_oVectorOfEnums);
+    byte _oSignedEnum = this.signedEnum();
+    _o.setSignedEnum(_oSignedEnum);
+    List<Integer> _oTestrequirednestedflatbuffer = new ArrayList<>();
+    for (int _j = 0; _j < this.testrequirednestedflatbufferLength(); ++_j) {_oTestrequirednestedflatbuffer.add(this.testrequirednestedflatbuffer(_j));}
+    _o.setTestrequirednestedflatbuffer(_oTestrequirednestedflatbuffer);
+    List<MyGame.Example.StatT> _oScalarKeySortedTables = new ArrayList<>();
+    for (int _j = 0; _j < this.scalarKeySortedTablesLength(); ++_j) {_oScalarKeySortedTables.add(this.scalarKeySortedTables(_j) != null ? this.scalarKeySortedTables(_j).unpack() : null);}
+    _o.setScalarKeySortedTables(_oScalarKeySortedTables);
+  }
+  public static int pack(FlatBufferBuilder builder, MonsterT _o) {
+    if (_o == null) return 0;
+    int _name = _o.getName() == null ? 0 : builder.createString(_o.getName());
+    int _inventory = 0;
+    if (_o.getInventory() != null) {
+      int[] __inventory = new int[_o.getInventory().size()];
```

Comment:
We really don't want to be allocating a temporary array here, but that would probably be fixed if it was a `int[]` to start with.